src/components/custom-constellation/template/HMRC_ODX_GDSTaskListTemplate/index.tsx

```diff
@@ -66,18 +66,9 @@ function HmrcOdxGdsTaskListTemplate(props: HmrcOdxGdsTaskListTemplateProps) {
         {children[0]}
       </Grid>
       <div>
-        <Text>{completedSections < totalSections ? 'Claim incomplete' : 'Claim complete'}</Text>
-        <br />
-        <br />
-        <Text variant='secondary'>
-          {`You have completed ${completedSections} of ${totalSections} sections.`}
-        </Text>
-        <br />
-        <br />
         <SummaryList
-          name='Tasks'
```

Comment:
this may need welsh translation

Reply:
Good spot - this component is in the Constellation side (Adviser).  Do we need to support Welsh in AUI?

Reply:
Yeah I don't think we need that on the constellation side. from my understanding it's for internal use!